Auto rerun on first error (#6620)
PipedreamHQ/pipedream: docs/docs/workflows/settings/README.md

If the step execution succeeds during any retry, the execution will proceed to the next step of the workflow.

> If the step fails on all 8 retries, and throws a final error, you'll receive [an error notification](/workflows/errors/) through your standard notification channel.

Author: ```suggestion
If the step fails on all 8 retries and throws a final error, you'll receive [an error notification](/workflows/errors/) through your standard notification channel.
```

### Send error notifications on the first error

> By default, if a step fails on all 8 retries, and throws a final error, you'll receive [an error notification](/workflows/errors/) through your standard notification channel. But sometimes you need to investigate errors as soon as they happen. If you're connecting to your database, and receive an error that the DB is down, you may want to investigate that immediately.

Author: ```suggestion
By default, if a step fails on all 8 retries and throws a final error, you'll receive [an error notification](/workflows/errors/) through your standard notification channel. But sometimes, you need to investigate errors as soon as they happen. If you're connecting to your database and receive an error that the DB is down, you may want to investigate that immediately.
```

Author: Removed comma before 'and' in a compound predicate; Added comma after introductory phrase

> On any workflow with auto-rerun enabled, you can optionally choose to **Send notification on first error**. This is disabled by default so you don't get emails for transient errors, but you can enable for critical workflows where you want visibility into all errors.

Author: ```suggestion
On any workflow with auto-rerun enabled, you can optionally choose to **Send notifications on the first error**. This is disabled by default, so you don't get emails for transient errors, but you can enable it for critical workflows where you want visibility into all errors.
```

Author: Added comma after 'by default'; Changed 'for' to 'it for' for clarity

> On any workflow with auto-rerun enabled, you can optionally choose to **Send notification on first error**. This is disabled by default so you don't get emails for transient errors, but you can enable for critical workflows where you want visibility into all errors.

Author: ```suggestion
On any workflow with auto-rerun enabled, you can optionally choose to **Send notifications on the first error**. This is disabled by default, so you don't get emails for transient errors, but you can enable it for critical workflows where you want visibility into all errors.
```

Author: Changed 'Send notification on first error' to 'Send notifications on the first error' for consistency

> For custom control over error handling, you can implement error logic in code steps (e.g. `try` / `catch` statements in Node.js code), or [create your own custom error workflow](/workflows/errors/#process-errors-with-custom-logic-instead-of-email).

Author: ```suggestion
For custom control over error handling, you can implement error logic in code steps (e.g., `try` / `catch` statements in Node.js code) or [create your own custom error workflow](/workflows/errors/#process-errors-with-custom-logic-instead-of-email).
```

Author: Added comma after 'e.g.'
